cmd/utils.go

```diff
@@ -2322,3 +2322,39 @@ func RemoveIfExists(path string) error {
 	}
 	return nil
 }
+
+func generateJSON(log *LoggingConfig, indexYaml IndexYaml, indexFilePath string) error {
```

Comment:
We could rename this function to generateCodewindJSON, because the format is specific to them.